lib/chef_zero/rspec.rb

```diff
@@ -227,9 +227,16 @@ def cookbook(name, version, data = {}, options = {}, &block)
         end
       end
 
-      def cookbook_artifact(name, identifier, data, &block)
+      def cookbook_artifact(name, identifier, data = {}, &block)
         with_object_path("cookbook_artifacts/#{name}") do
-          ChefZero::RSpec.server.load_data({ 'cookbook_artifact' => { name => { identifier => data } } }, current_org)
+          if data.has_key?('metadata.rb')
+            if data['metadata.rb'].nil?
+              data.delete('metadata.rb')
```

Comment:
Not having metadata.rb means you want one (since you almost always do); specifying it `nil` means you *don't* want one (rare but possible). This is a cargo cult from the cookbooks load_data, which I think we want to be identical.